Add Trino tenth birthday post (#325)
trinodb/trino.io: _posts/2022-08-08-trino-tenth-birthday.md

---
layout: post

> title: "Happy 10th Birthday Trino!"

Reviewer: ```suggestion
title: "Happy tenth birthday Trino!"
```

author: "Brian Olsen, Martin Traverso, Dain Sundstrom, David Phillips, Eric Hwang"
excerpt_separator: <!--more-->
---

![](/assets/blog/trino-tenth-birthday/how-it-started-going.png)

It’s inspiring and mindblowing to reflect on the ten year journey that has
produced the community around Trino. Trino is the community-driven fork from

> Presto, the distributed big data query engine created at Facebook in 2012. We

Reviewer: ```suggestion
Presto, the distributed big data SQL query engine created at Facebook in 2012. We
```

are a community of engineers, scientists, analysts, and visionaries that work in
a fast paced world where the expectations on the time to insights from our
analytics and the scale of the data are ever-increasing. Sometimes words only do
so much justice to encompass a journey like this one, so we created a video to
let you experience it yourself! Enjoy!

<!--more-->

# Trino’s first ten years video

<iframe src="https://youtu.be/hPD95_-bZZw" width="800" height="500" frameborder="0" marginwidth="0" marginheight="0" scrolling="no" style="border:1px solid #CCC; border-width:1px; 
margin-bottom:5px; max-width: 100%;" allowfullscreen=""> 
</iframe>

As we watch the video and think back to the five years Presto and Trino shared,
you begin to appreciate the organic development of the community, and the
excitement around the solution space that the project brought to big data. As a
baseline, Trino offers a faster and more interactive alternative to accessing
data stored in HDFS via Hive. But the project didn’t stop there. Development of

> the SPI abstracted metadata and storage, among other interfaces to different

Reviewer: ```suggestion
the SPI abstracted metadata and storage access to different
```

> systems, making Trino a suitable system to query an entire data ecosystem from

Reviewer: ```suggestion
systems, making Trino a suitable engine to query an entire data ecosystem from
```

one location using ANSI SQL! Since the projects split, Trino has skyrocketed in

> development from the original project and has added an array of features that

Reviewer: ```suggestion
development from the original project and added an array of features that
```

> we’ve listed out in the evolution of the Trino architecture blog post.

Reviewer: link

![](/assets/blog/trino-tenth-birthday/trajectory.png)

To really celebrate this milestone, we wanted to offer some exciting ways for
you to learn more about Trino, and spin up Trino on your own system to play
around with it. We have a list of blogs, project stats, and ways to get involved

> below. Starburst is also celebrating by offering free Trino birthday t-shirts

Author: Add link from @monimiller when ready

when you complete their Space Quest League mission. Also don’t forget to attend 

> our annual Trino Summit in November!

Reviewer: link to blog post on trino.io

* [Trino getting started repository](https://github.com/bitsondatadev/trino-getting-started)

# Community statistics

* 28250+ commits 💻 in GitHub
* 5750+ stargazers ⭐ in GitHub
* 7350+ members 👋 in Slack
* 6950+ pull requests merged ✅ in GitHub
* 4000+ issues 📝 created in GitHub
* 3750+ followers 🐦 on Twitter
* 650+ average weekly members 💬 in Slack
* 1050+ subscribers 📺 in YouTube
* 38 Trino Community Broadcast ▶️ episodes
* 264 Presto + Trino 🚀 releases (not including PrestoDB releases since the 
fork)

# Join our community

* Join the [Trino Slack workspace]({{site.base}}/slack.html)
* Watch the [Trino Community Broadcast]({{site.base}}/broadcast/)
* Subscribe to the [Trino YouTube channel](https://www.youtube.com/c/trinodb)
* Follow us on the [trinodb Twitter account](https://twitter.com/trinodb)
* Give us a star on the [Trino GitHub repository](https://github.com/trinodb/trino)
* Follow us on the [Trino LinkedIn account](https://www.linkedin.com/company/trino-software-foundation)

# Trino Summit 2022

We hope you all join us in celebrating Trino’s birthday today. If you want to 
learn even more, 

> [sign up for our hybrid event Trino Summit November 10th, 2022](https://www.starburst.io/info/trinosummit/).

Reviewer: ```suggestion
[sign up for our hybrid event, Trino Summit, on the 10th of November 2022](https://www.starburst.io/info/trinosummit/).
```
